docs/examples.md

```diff
@@ -281,6 +282,70 @@ CREATE STREAM pageviews_interest_contact AS \
   FROM pageviews_enriched;
 ```
 
+<a name="avro"></a>
+### Avro format and Schema Registry Integration
+
+KSQL can read from and write into kafka topics with avro format for the message values where the
+avro schemas are managed by the Schema Registry service. To read message values that are in
+avro format KSQL will automatically fetch the avro schema from the Schema Registry service and
+reads the message. When writing messages into the sink topics, KSQL will register the avro schema
+ of the message value into the Schema Registry service.
+
+Note that KSQL considers the message keys in kafka topics are in STRING format and regardless
+of the format that the message key has, KSQL will read it as a String. Therefore, KSQL will
+ignore any schema on the message key in kafka topics with avro format.
+
+ In order to use Schema Regirsty in KSQL you need to provide the url for the Schema Registry
+ service to the KSQL engine. You can do this by setting the `ksql.schema.registry.url` property
+ in the properties file that you use to start the KSQL engine. Here is an example:
+
+ `ksql.schema.registry.url=http://myschemaregistry.com:8081`
+
+ If the `ksql.schema.registry.url` property is not set explicitly KSQL will use the default value
```

Comment:
If the schema registry service is not available only avro support will be unavailable and it won't affect anything else.

Reply:
And this should also be explained in our docs. :-)

Reply:
Also, we should clarify this if possible / if it isn't too complicated to write down.  For example, if S.R. becomes unavailable, KSQL can continue processing with any locally cached Avro schemas (= schemas it has already seen before S.R. went offline), but it will not be able to parse message with schemas it hasn't seen yet.

Reply:
Also: Is it possible for the user to NOT specify `ksql.schema.registry.url` during startup (via properties) and instead, later on, `SET` the config in the CLI?  Or *must* the url be set prior to KSQL CLI startup?